components/Pages/Login/LoginForm.tsx

```diff
@@ -3,75 +3,99 @@ import Router from 'next/router'
 import { useForm } from 'react-hook-form'
 import { ToastContainer } from 'react-toastify'
 import { AxiosError } from 'axios'
+import classNames from 'classnames'
 
 import login from 'lib/api/login'
 import Auth from 'lib/auth'
 
 import Button from 'components/Button'
 
 export interface Data {
-	email: string
-	password: string
+  email: string
+  password: string
 }
 
 export default function LoginForm() {
-	const { register, handleSubmit, errors } = useForm()
+  const { register, handleSubmit, errors } = useForm()
 
-	const [isSubmitting, setIsSubmitting] = useState<boolean>(false)
-	const onSubmit = async ({ email, password }: Data) => {
-		try {
-			setIsSubmitting(true)
-			const token = await login.submit(email, password)
-			Auth.login(token, email)
-			Router.push('/clases')
-		} catch (error) {
-			setIsSubmitting(false)
-			login.errorHandler(error as AxiosError)
-		}
-	}
+  const [isSubmitting, setIsSubmitting] = useState<boolean>(false)
+  const onSubmit = async ({ email, password }: Data) => {
+    try {
+      setIsSubmitting(true)
+      const token = await login.submit(email, password)
+      Auth.login(token, email)
+      Router.push('/clases')
+    } catch (error) {
+      setIsSubmitting(false)
+      login.errorHandler(error as AxiosError)
+    }
+  }
 
-	return (
-		<form
-			onSubmit={handleSubmit(onSubmit)}
-			className='login-form'
-		>
-			<ToastContainer position='top-center' />
-			<label className='label label-login'>
-				Usuario
-			</label>
-			<input
-				className='input input-login'
-				type='email'
-				name='email'
-				placeholder='usuario@ejemplo.com'
-				ref={register({ required: true })}
-			/>
-			{errors.email &&
-				<span className='error help is-danger is-medium'>
-					Este campo es necesario
-				</span>}
-			<label className='label label-login'>
-				Contraseña
-			</label>
-			<input
-				className='input input-login'
-				type='password'
-				name='password'
-				placeholder='contraseña'
-				ref={register({ required: true })}
-			/>
-			{errors.password &&
-				<span className='error help is-danger is-medium '>
-					Este campo es necesario
-				</span>}
-			<div className='mt-14'>
-				<Button
-					isPrimary
-					label={isSubmitting ? 'Enviando' : 'Ingresar'}
-					type='submit'
-					isDisabled={isSubmitting}
-				/>
-			</div>
-		</form>
-	)
+  return (
+    <form
+      onSubmit={handleSubmit(onSubmit)}
+      className='flex flex-col w-full'
+    >
+      <ToastContainer position='top-center' />
+      <label className='mt-9 mb-5'>
+        Usuario
+      </label>
+      <div className='flex flex-col relative'>
+        <input
+          className={classNames(
+            'bg-black/70 text-brand-complementary',
+            'rounded-small py-1 px-3',
+            'focus:border focus:border-solid focus:border-brand-primary',
```

Comment:
```suggestion
            'focus:outline focus:outline-1 focus:outline-brand-primary',
```
- Me parece que puedes aplicar esto 👆🏻 para que no se vea ese color azulito en los inputs al hacerles focus
- Así tampoco se moverían los inputs y sus labels 2 px hacia abajo cuando al hacerles focus se agrega el border